include/psa/crypto_types.h

```diff
@@ -380,4 +380,54 @@ typedef uint16_t psa_key_derivation_step_t;
 
 /**@}*/
 
+/** \defgroup pake Password-authenticated key exchange (PAKE)
+ * @{
+ */
+
+/** \brief Encoding of the side of PAKE
+ *
+ * Encodes which side of the protocol is being executed. For more information
+ * see the documentation of individual PSA_PAKE_SIDE_XXX constants.
+ */
+typedef uint16_t psa_pake_side_t;
```

Comment:
Is there a reason for making this a 16-bit instead of an 8-bit type? I see the `PSA_PAKE_SIDE_XXX` constants really only use 2 bits of information at the moment: whether the algorithm is augmented, and whether the 'side' is client/first or server/second

Reply:
Good point, `uint8_t` should be enough.